### ISSUE TITLE
Probe: log crate

### ISSUE THREAD

**Issue by goodhoko** [tonarino/acoustic_profiler#8]

**Comment by PabloMansanet** [comment]
Implementing a logger is pretty simple; here's the example code provided in the docs:

```rust
use log::{Record, Level, Metadata};

struct SimpleLogger;

impl log::Log for SimpleLogger {
    fn enabled(&self, metadata: &Metadata) -> bool {
        metadata.level() <= Level::Info
    }

    fn log(&self, record: &Record) {
        if self.enabled(record.metadata()) {
            println!("{} - {}", record.level(), record.args());
        }
    }

    fn flush(&self) {}
}
```
It should be easy to create a logger that turns these logged records into events to send to the aggregator.

The specifics will depend on what we define those events to look like, but here are some general ideas on what things we could be watching out for to turn into sound:

* Frequency of messages of each log level -> frequency of some waves. Clean, nice sine waves for info and debug, but perhaps something a bit more discordant for warnings and errors?
* Specific clicks for keywords detected on the logs.
* ... TBD

**Comment by goodhoko** [comment]
> The specifics will depend on what we define those events to look like, but here are some general ideas on what things we could be watching out for to turn into sound: [...]

Yeah, these all sound good. Supposing the event as proposed in #11, the probe could send an event for each log with a quality set based on the log-level: low for debug and high for errors and then pair it up with a sound effect that varies volume based on event frequency and goes from a sine wave to square wave based on the quality.
